frontend_lib/src/tinymceAutoCompleteHelper.js

```diff
@@ -0,0 +1,101 @@
+const atSymbolCode = '\ufeff'
+
+export const tinymceAutoCompleteHandleInput = (event, tinymcePosition, setState, fetchMentionList, isAutoCompleteActivated) => {
+  if (!event.data) return
+
+  switch (event.data) {
+    case '@': {
+      const rawHtml = `<span id="autocomplete"><span id="autocomplete__searchtext"><span id="autocomplete__start">${atSymbolCode}</span></span></span>`
+      const currentTextContent = tinymce.activeEditor.selection.getStart().textContent
+      const hasSpaceBeforeAt = currentTextContent.length === 1 || currentTextContent[currentTextContent.length - 2] === ' '
```

Comment:
Since this constant also checks if the @ is at the beginning of the text, I would change the name to `isAtTheBeginningOrHasSpaceBefore` or try something more simple like `isTheBeginningOfAMention`